src/lib/types.ts

```diff
@@ -48,8 +48,12 @@ export interface Vessel {
 }
 
 export interface Owner {
+  beaconOwnerFullName: string;
+  beaconOwnerEmail?: string;
+  beaconOwnerTelephoneNumber?: string;
+  beaconOwnerAlternativeTelephoneNumber?: string;
   beaconOwnerAddressLine1: string;
-  beaconOwnerAddressLine2?: string;
+  beaconOwnerAddressLine2: string;
```

Comment:
I think this change makes `beaconOwnerAddressLine2` mandatory?  But it hasn't effected the UI.  What was the reason behind removing the `?`?

Reply:
I don't think it will make it mandatory in terms of validation if that's what you're referring to? It's not in the lookup of validators in the file `ownerFieldValidators.ts` so will not be validated.

Looking at the `types.ts` file we haven't identified any other optional fields using `?`, being consistent we should either remove `?` from the type definitions and let the validation handle it, or add the optional flag `?` to all the types?